scripts/deployConfigs.ts

```diff
@@ -15,18 +15,16 @@ const networkConfigs: Record<number, Config> = {
     investPct: 9000, // 90%
     perfFeePct: 300, // TODO
     multisig: "0x035F210e5d14054E8AE5A6CFA76d643aA200D56E",
-    minLockPeriod: 60 * 60 * 24 * 30, // 30 days
-    ethAnchorRouter: "0xcEF9E167d3f8806771e9bac1d4a0d568c39a9388",
+    minLockPeriod: 60 * 60 * 24 * 14, // 14 days
     AUstToUstPriceFeed: "0x7b80a92f7d1e5cEeDDf939d77BF281E7e88f2906",
   },
 
-  // rinkeby
-  4: {
+  // ropsten
+  3: {
     investPct: 9000, // 90%
     perfFeePct: 100, // 1%
     multisig: "deployer",
     minLockPeriod: 1, // 1 second
-    ethAnchorRouter: "0x7537aC093cE1315BCE08bBF0bf6f9b86B7475008",
     AUstToUstPriceFeed: "TODO",
```

Comment:
There is no AUstToUstPriceFeed on ropsten network. So we need to use ExchangeRate contract.
There are TestAnchorUstStrategy and TestAnchorNonUstStrategy contract which inherited from AnchorUstStrategy and AnchorNonUstStrategy.
These strategies were made for ropsten testnet.
We cannot use AnchorUstStrategy and AnchorNonUstStrategy on ropsten.

Reply:
@ryuheimat I dont' understand

why don't we deploy `MockChainlinkPriceFeed` to solve that? don't think we need TestAnchor strategies for that

Reply:
Lol, why I couldn't think about that. EthAnchor was using uniswap to swap, so I just followed.